ui/src/components/directory/topbar/eclSearch/EclTree.vue

```diff
@@ -0,0 +1,397 @@
+<template>
+  <div class="flex flex-column justify-contents-start" id="ecl-tree-bar-container">
+    <div
+      v-if="dialogRef?.options?.data?.focus?.name && dialogRef.options.data.type"
```

Comment:
probably better to make both optional - or move this check into a method/computed in order to deal with undefined properties